deployment/group_vars/dai.yml

```diff
@@ -45,7 +46,9 @@ UI_FOREIGN_GAS_PRICE_FALLBACK: 10000000000
 MONITOR_PORT: 3003
 MONITOR_HOME_DEPLOYMENT_BLOCK: 759
 MONITOR_FOREIGN_DEPLOYMENT_BLOCK: 6478417
-MONITOR_GAS_LIMIT: 300000
-MONITOR_GAS_PRICE_FALLBACK: 21
+MONITOR_HOME_GAS_LIMIT: 300000
+MONITOR_FOREIGN_GAS_LIMIT: 300000
+MONITOR_HOME_GAS_PRICE_FALLBACK: 21
+MONITOR_FOREIGN_GAS_PRICE_FALLBACK: 21
```

Comment:
Should `MONITOR_FOREIGN_GAS_PRICE_FALLBACK` be equal `FOREIGN_GAS_PRICE_FALLBACK`?